libs/dh/market-participant/actors/feature-actors/src/lib/create/dh-actors-create-actor-modal.component.ts

```diff
@@ -0,0 +1,134 @@
+import { WattTextFieldComponent } from '@energinet-datahub/watt/text-field';
+import { WATT_MODAL, WattModalComponent } from '@energinet-datahub/watt/modal';
+import { DhDropdownTranslatorDirective } from '@energinet-datahub/dh/shared/ui-util';
+import { GetOrganizationsDocument } from '@energinet-datahub/dh/shared/domain/graphql';
+import { WattDropdownComponent, WattDropdownOptions } from '@energinet-datahub/watt/dropdown';
+import { dhCvrValidator, dhDomainValidator } from '@energinet-datahub/dh/shared/validators';
+
+@Component({
+  standalone: true,
+  selector: 'dh-actors-create-actor-modal',
+  templateUrl: './dh-actors-create-actor-modal.component.html',
+  styles: [
+    `
+      :host {
+        display: block;
+      }
+
+      .dh-actors-create-actor-organization {
+        vater-stack {
+          width: 50%;
+        }
+
+        vater-stack[direction='row'] {
+          watt-dropdown,
+          watt-text-field {
+            width: 50%;
+          }
+        }
+      }
+
+      .dh-actors-create-actor-modal__form {
+        watt-dropdown {
+          width: 50%;
+        }
+      }
+    `,
+  ],
+  imports: [
+    TranslocoDirective,
+    ReactiveFormsModule,
+    NgIf,
+
+    WATT_MODAL,
+    WATT_STEPPER,
+    WattDropdownComponent,
+    WattButtonComponent,
+    WattTextFieldComponent,
+    WattFieldErrorComponent,
+    DhDropdownTranslatorDirective,
+    VaterStackComponent,
+  ],
+})
+export class DhActorsCreateActorModalComponent {
+  private _fb: NonNullableFormBuilder = inject(NonNullableFormBuilder);
+  private _apollo = inject(Apollo);
+
+  private _getOrganizationsQuery$ = this._apollo.watchQuery({
+    useInitialLoading: true,
+    notifyOnNetworkStatusChange: true,
+    query: GetOrganizationsDocument,
+  });
+
+  @ViewChild(WattModalComponent)
+  innerModal: WattModalComponent | undefined;
+
+  organizationOptions: WattDropdownOptions = [];
+  countryOptions: WattDropdownOptions = [
+    { value: 'DK', displayValue: 'DK' },
+    { value: 'SE', displayValue: 'SE' },
+    { value: 'NO', displayValue: 'NO' },
+    { value: 'FI', displayValue: 'FI' },
+  ];
+
+  chooseOrganizationForm = this._fb.group({ orgId: [''] });
+  newOrganizationForm = this._fb.group({
+    country: ['', Validators.required],
+    cvrNumber: ['', [Validators.required, dhCvrValidator()]],
+    companyName: ['', Validators.required],
+    domain: ['', [Validators.required, dhDomainValidator]],
+  });
+
+  showCreateNewOrganization = false;
+
+  constructor() {
+    this._getOrganizationsQuery$.valueChanges.pipe(takeUntilDestroyed()).subscribe((result) => {
+      if (result.data?.organizations) {
+        this.organizationOptions = result.data.organizations.map((org) => ({
+          value: org.organizationId ?? '',
+          displayValue: org.name,
+        }));
+      }
+    });
+  }
+
+  toggleShowCreateNewOrganization(): void {
+    this.showCreateNewOrganization = !this.showCreateNewOrganization;
```

Comment:
suggestion: consider resetting the `newOrganizationForm` group when `showCreateNewOrganization` is set to `false`, so the user sees an empty form if the step is shown again.